src/base/preferences.cpp

```diff
@@ -438,6 +438,16 @@ void Preferences::setActionOnDblClOnTorrentFn(int act)
     setValue("Preferences/Downloads/DblClOnTorFn", act);
 }
 
+bool Preferences::getEditableSavePath() const
+{
+    return value("Preferences/Downloads/EditableSavePath").toBool();
+}
+
+void Preferences::setEditableSavePath(bool enabled)
```

Comment:
setEditableSavePath**Enabled**(bool enabled)? It's a little ambiguous without it as if expecting a string argument.